src/dom/Value.ts

```diff
@@ -309,6 +343,99 @@ export function Value<Clazz extends Constructor>(
       this._unsupportedOperation("deleteField");
     }
 
+    /**
+     * Helper method to be implemented by each dom.Value with individual equivalence logic.
+     *
```

Comment:
I think this text is borrowed directly from the Ion spec. If so, I suggest adding a link to that section of the spec instead of including the text inline here. That way if the spec is updated to clarify something, we won't have to update it here as well.